src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "Microsoft.AspNetCore.Hosting": "1.0.0-*",
     "Microsoft.AspNetCore.Server.WebListener": "0.1.0-*",
-    "Microsoft.CodeAnalysis.CSharp": "1.1.0-rc1-20151109-01",
+    "Microsoft.CodeAnalysis.CSharp": "1.2.0-beta1-20160202-02",
```

Comment:
Curious, isn't `Microsoft.CodeAnalysis.CSharp` pulled in transitively from `Microsoft.Extensions.CodeGeneration.Templating`?